### ISSUE TITLE
How to make Fading effect appear automatically when Listview is scrollable?

### ISSUE THREAD

**Issue by trantruongan136** [mponkin/fading_edge_scrollview#19]
I am using a SingleChildScrollView and inside is an AnimatedList that can automatically scroll down to the last item. I found that the effect only appeared when I was using my hand and scrolling. I want to know is there a way to make the Fading effect appear automatically when the Listview is scrollable?

**Comment by CoolDude53** [comment]
@trantruongan136 I made a fork that implements this. If you pass through the child scrollview's list to the FadingEdgeScrollView with the new `scrollViewList` paramenter, the FadingEdgeScrollView will update anytime the underlying list changes. You can use this until this functionality is properly added to the plugin.

To import in your pubspec.yaml use:
```
 fading_edge_scrollview:
    git:
     url: https://github.com/CoolDude53/fading_edge_scrollview.git
     ref: master
```

**Comment by mponkin** [comment]
fixed in version 4.0.0
